code/datums/mind.dm

```diff
@@ -343,7 +343,7 @@ datum/mind
 			switch(href_list["common"])
 				if("undress")
 					for(var/obj/item/W in current)
-						current.drop_inv_item_on_ground(W)
+						current.dropItemToGround(W)
```

Comment:
this is not in your list of changes, why are you changing the names of inventory procs?

Reply:
actually this is in the list of changes.
It's to save up brain matter from having to memorize different names for the same procs by the by.